tailwind.config.js

```diff
@@ -3,6 +3,37 @@ module.exports = {
   content: ['./src/pages/**/*.{js,jsx,ts,tsx}', './src/components/**/*.{js,jsx,ts,tsx}'],
   theme: {
     extend: {},
+
+     fontSize: {
+      H2 : ['40px','48px'],
```

Comment:
do not config font sizes, use the normal tailwind ones: https://tailwindcss.com/docs/font-size
Do not config things that aren't necessary at all.